### PR TITLE
Add times to Number prototype

### PR DESCRIPTION
**Pull request by stg101**
Add times method to Number prototype
Usage :
new Number(2).times(console.log);

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 const p = console.log;
 const puts = console.log;
 
+Number.prototype.times = function(callback) {
+  for (let i = 0; i < +this; i++) {
+    callback(i);
+  }
+  return;
```
Comment (cristianbgp) [inline src/index.js]: I think that if you're returning `undefined` you could leave it. A function always return `undefined` by default.

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+};
+
+// usage :
+// new Number(2).times(console.log);
```
Comment (cristianbgp) [inline src/index.js]: I think the usage of this function should be something like this
```
(2).times( () => console.log("Hello") );
```
Also, I'm thinking about adding a file with examples of usage, so please delete the commented lines

File: src/index.js (hunk 1 above)
Comment (stg101) [reply src/index.js]: Thanks, I'll change that

File: src/index.js (hunk 2 above)
Comment (stg101) [reply src/index.js]: Thanks, I'll change that

**Pull request merged**